src/typeahead/typeahead.js

```diff
@@ -116,6 +116,18 @@ angular.module('ui.bootstrap.typeahead', ['ui.bootstrap.position', 'ui.bootstrap
           element.removeAttr('aria-activedescendant');
         } else {
           element.attr('aria-activedescendant', getMatchId(index));
+        
+        
+          var activeLiElem = document.getElementById(getMatchId(index));
+          if(activeLiElem) {
+              var activeAElem = activeLiElem.getElementsByTagName('A')[0];
```

Comment:
This makes too much assumption about the presence of an anchor tag in this matcher. Need to guard against no result being returned.